src/01-functions.js

```diff
@@ -74,7 +86,10 @@ function multiply() {}
  *  average(10, 6);
  *  //> 8 // 10 + 6 = 16; 16/2 = 8;
  */
-function average() {}
+function average(a, b) {
+  resultAver = ((a + b)/2);
```

Comment:
We do not need the outer `()` here.  It won't break the code but to leave it but we should know it is not necessary